Builtin/LUIInputField.py

```diff
@@ -51,6 +52,7 @@ def __init__(self, parent=None, width=200, placeholder=u"Enter some text ..", va
             self.parent = parent
 
         LUIInitialState.init(self, kwargs)
+        self.re_skip = re.compile("\W*\w+\W")
```

Comment:
Smaller issue, the regexp should probably be a class attribute instead of an instance attribute, there is no need to store it per instance